Documentation/kubernetes-networking.md

```diff
@@ -58,11 +59,12 @@ The following requirements must be met by your existing infrastructure to use Te
 
 Each pod in the Kubernetes cluster will be assigned an IP that is expected to be routable from all other hosts and pods in the Kubernetes cluster.
 
-The general model for achieving this is to first assign an IP range to each host in your cluster.
+An easy way to achieve this is to use Calico. The Calico agent is already running on each node to enforce network policy. Starting it with the `CALICO_NETWORKING` environment variable set to `true` will cause it to run a BGP agent inside the Calico agent pod. These BGP agents will automatically form a full mesh network to exchange routing information. This allows a single large IP range to be used across your whole cluster and IP addresses to be efficiently assigned from it.  To peer with your existing BGP infrastructure follow this [guide][calico-bgp]. If you're Kubernetes clusters is hosted on an [L2 network](calico-l2) (e.g. in your own datacenter or on AWS) there is no need to peer with your routers.
```

Comment:
The calico-l2 link here should be `[L2 network][calico-l2]` .. (note the square brackets)